github/localrepo.go

```diff
@@ -191,14 +191,14 @@ func (r *GitHubRepo) MainRemote() (remote *Remote, err error) {
 func (r *GitHubRepo) MainProject() (project *Project, err error) {
 	origin, err := r.MainRemote()
 	if err != nil {
-		err = fmt.Errorf("Aborted: the origin remote doesn't point to a GitHub repository.")
+		err = fmt.Errorf("Aborted: the origin remote doesn't point to a GitHub repository: %v.", err)
```

Comment:
Since this error message will get displayed on a very long line, and doesn't feel like a single sentence anymore, how about breaking it up into two lines?

```
Aborted: the origin remote doesn't point to a GitHub repository.
Invalid GitHub URL: ssh://mygh/Manganeez/repo.git
```